sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py

```diff
@@ -193,7 +193,7 @@ def handle_port_update_event(self):
                                                             db_name,
                                                             table_name)
                 # This is the final event considered for processing
-                self.logger.log_warning("*** {} handle_port_update_event() fvp {}".format(
+                self.logger.log_info("*** {} handle_port_update_event() fvp {}".format(
```

Comment:
@bradh352 can you please keep this log_notice() because I want to know which event Xcvrd finally consumed.

Reply:
done